src/framework/Elsa.Studio.Core/Elsa.Studio.Core.csproj

```diff
@@ -33,12 +33,12 @@
         <PackageReference Include="ThrottleDebounce"/>
     </ItemGroup>
 
-<!--    <ItemGroup Label="Elsa">-->
-<!--        <ProjectReference Include="..\..\..\..\..\elsa-core\main\src\clients\Elsa.Api.Client\Elsa.Api.Client.csproj"/>-->
-<!--    </ItemGroup>-->
-
     <ItemGroup Label="Elsa">
-        <PackageReference Include="Elsa.Api.Client"/>
+        <ProjectReference Include="..\..\..\..\..\elsa-core\main\src\clients\Elsa.Api.Client\Elsa.Api.Client.csproj"/>
```

Comment:
How is this going to work?

Reply:
It won't; I am waiting for the elsa-core PR to be completed & built, so that I can update this solution to point to a newer version of Elsa.Api.Client.